### DISCUSSION TITLE
Sector Generator: Mark Systems that are interesting

### DISCUSSION THREAD

**Discussion by Gegh** [bcholmes/StarTrek2d20#179] [Ideas]
As a GM I want to create a whole Sector but for our campaign only the POI are relevant. So I would like to have the option to only create Sectors/Systems with at least on POI. Maybe even define that POI.

**Comment by Felderburg** [comment]
I'm pretty sure that is what the tool does by default. It has "Notable Systems" above the list when a sector is generated, so I assume that's what it's doing. The rules as written do so anyways, and note that sectors are assumed to be filled with other, uninteresting, locations that aren't generated.

**Reply by bcholmes** [reply]
I suspect that some of the ask might relate to ensuring that there's, say, a nice class-M world with life in the system, rather than a barren desert world (which can be an output of the random process). And being able to specify some of those choices (e.g. "I need a jungle-covered class M planet") would be valuable.
